scripts/generate-report.py

```
@@ -433,32 +954,89 @@ def build_executive_summary(doc, df, epoch, stats, date_label):
+    set_cell_shading(cell, CARD_BG)
+    set_cell_margins(cell, top=120, bottom=120, left=160, right=160)
+
+    epoch_id = epoch.get("id", "?")
+    created_at = epoch.get("created_at", "unknown")
+    total_posts = format_int(stats.get("total_posts", "?"))
+    scored_count = format_int(stats.get("scored_count", "?"))
+    last_24h = format_int(stats.get("last_24h", "?"))
+    gen_time = datetime.now(timezone.utc).strftime("%Y-%m-%d %H:%M UTC")
+
+    lines = [
+        f"Scope: Top 1,000 scored posts from the active governance epoch.",
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)